src/ngraph_rewrite_pass.cc

```diff
@@ -231,7 +231,9 @@ class NGraphEncapsulationPass : public NGraphRewritePass {
     }
 
     // 4. Encapsulate clusters then, if requested, dump the graphs.
-    TF_RETURN_IF_ERROR(EncapsulateClusters(options.graph->get(), idx));
+    FunctionDefLibrary* fdeflib_new = new FunctionDefLibrary();
+    TF_RETURN_IF_ERROR(
+        EncapsulateClusters(options.graph->get(), idx, fdeflib_new));
```

Comment:
Note, we are populating the flib in normal path, but not using it. Only grappler path is using flib